src/main/features/claim/views/eligibility/government-department.njk

```diff
@@ -1,16 +1,18 @@
 {% extends "layout.njk" %}
 {% from "form.njk" import csrfProtection, errorSummary, radioGroup, saveAndContinueButton %}
+{% from "externalLink.njk" import externalLink %}
 
 {% set title = t('Govornment department | Money claim') %}
 
+{% set governmentDepartmentUri = 'https://www.gov.uk/government/publications/serve-the-treasury-solicitor-with-legal-proceedings' %}
+
 {% block content %}
 
   <div class="grid-row">
     <div class="column-two-thirds">
       <h1 class="heading-large">{{ t('Are you claiming against a government department?') }}</h1>
       <p>
-        {{ t('<a href="{{ governmentDepartmentUri }}" target="_blank" >See a list of government departments</a>.',
-        { governmentDepartmentUri : 'https://www.gov.uk/government/publications/serve-the-treasury-solicitor-with-legal-proceedings' }) | safe }}
+        {{ t(externalLink('See a list of government departments', governmentDepartmentUri) + '.') | safe }}
```

Comment:
this causes the link to be translated, we should just be translating the text,
`t('See a list of government departments')`